mobile-widgets/qmlmanager.cpp

```diff
@@ -1436,14 +1434,9 @@ QStringList QMLManager::buddyInit() const
 	int i = 0;
 	for_each_dive (i, d) {
 		QString temp = d->buddy;
-		if (!temp.isEmpty() && !temp.contains(",")){
+		if (!temp.isEmpty()){
 			buddies << d->buddy;
 		}
```

Comment:
this now pushes the string with the commas into buddies (which is a QStringList) - that's not what you want, is it?
See two comments below